src/components/standfirst.tsx

```diff
@@ -67,6 +67,23 @@ const immersive = (pillarStyles: PillarStyles): SerializedStyles => css`
     }
 `;
 
+const media = css`
+    color: ${neutral[86]};
+    p, ul, li {
+        line-height: 20px;
```

Comment:
Could we use the design system for line height?

`body.medium({ lineHeight: 'loose' })`